_data/collection-index.yml

```
@@ -613,3 +613,7 @@
   contact: https://github.com/MarlonPassos-git/dev-container-features/issues
   repository: https://github.com/MarlonPassos-git/dev-container-features
   ociReference: ghcr.io/marlonpassos-git/dev-container-features
+- name: DevContainers for Fedora-based Development
+  maintainer: RonaldTM9999
+  contact: https://github.com/RonaldTM9999/devcontainers/issues
+  ociReference: https://ghcr.io/RonaldTM9999/devcontainers
```

```suggestion
  ociReference: ghcr.io/ronaldtm9999/devcontainers
```

Hi 👋

The ORAS spec requires the ociReference to be in lower case. Can we update here along with the removal of `https` prefix, thanks?
The supporting tools do help convert the case, however, its better to sync as much as possible.